config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -4233,6 +4233,17 @@
     </details>
   </checkerFrameworkError>
 
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java</fileName>
+    <specifier>return</specifier>
+    <message>incompatible types in return.</message>
+    <lineContent>return blockTag;</lineContent>
+    <details>
+      type of expression: @Initialized @Nullable String
+      method return type: @Initialized @NonNull String
+    </details>
+  </checkerFrameworkError>
```

Comment:
had to add cuz ci was failing::

https://github.com/checkstyle/checkstyle/actions/runs/11137849799/job/30951942392?pr=15710

Reply:
please remove this and suppression and just add annotation `@Nullable` over class  to make it clear that you return null potentially.